packages/graphql/src/schema/validation/utils/map-error.test.ts

```diff
@@ -0,0 +1,311 @@
+                    "Invalid argument: filter, error: Int cannot represent non-integer value."
+                );
+            });
+            test("Float", () => {
+                const error = new GraphQLError(
+                    "Invalid argument: filter, error: Float cannot represent non numeric value: false",
+                    {
+                        nodes: [argumentNode, directiveNode],
+                        extensions: { exception: { code: VALIDATION_ERROR_CODES.AUTHORIZATION } },
+                    }
+                );
+                const mappedError = mapError(error);
+                expect(mappedError).toHaveProperty("message");
+                expect(mappedError.message).toBe(
+                    "Invalid argument: filter, error: Float cannot represent non numeric value."
+                );
+            });
+            test("String", () => {
+                const error = new GraphQLError(
+                    "Invalid argument: filter, error: String cannot represent non string value: 2",
+                    {
+                        nodes: [argumentNode, directiveNode],
+                        extensions: { exception: { code: VALIDATION_ERROR_CODES.AUTHORIZATION } },
+                    }
+                );
+                const mappedError = mapError(error);
+                expect(mappedError).toHaveProperty("message");
+                expect(mappedError.message).toBe(
+                    "Invalid argument: filter, error: String cannot represent non string value."
+                );
+            });
+            test("Boolean", () => {
+                const error = new GraphQLError(
+                    "Invalid argument: filter, error: Boolean cannot represent non boolean value: 1",
+                    {
+                        nodes: [argumentNode, directiveNode],
+                        extensions: { exception: { code: VALIDATION_ERROR_CODES.AUTHORIZATION } },
+                    }
+                );
+                const mappedError = mapError(error);
+                expect(mappedError).toHaveProperty("message");
+                expect(mappedError.message).toBe(
+                    "Invalid argument: filter, error: Boolean cannot represent non boolean value."
+                );
+            });
+        });
+        test("enum value does not exist in enum", () => {
+            const error = new GraphQLError(
+                'Invalid argument: filter, error: Value "CREATE" does not exist in "AuthorizationValidateOperation" enum.',
+                {
+                    nodes: [argumentNode, directiveNode],
+                    extensions: { exception: { code: VALIDATION_ERROR_CODES.AUTHORIZATION } },
+                }
+            );
+            const mappedError = mapError(error);
+            expect(mappedError).toHaveProperty("message");
+            expect(mappedError.message).toBe('Invalid argument: filter, error: Value "CREATE" does not exist in enum.');
+        });
```

Comment:
This doesn't feel quite right, given that `AuthorizationValidateOperation` is a static enum which we could document?

Reply:
I wouldn't start discriminating between the mystery types and only erase some while keeping others. We could've gone all the way with documenting the validation types and used that approach for all..

Reply:
added some comments